bodhi/client/__init__.py

```diff
@@ -261,18 +267,18 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-def _save_override(url, user, password, staging, edit=False, openid_api=None, **kwargs):
+def _save_override(url: str, user: str, password: str, staging: bool, edit: bool = False,
+                   openid_api: typing.Optional[str] = None, **kwargs):
     """
     Create or edit a buildroot override.
 
     Args:
-        url (unicode): The URL of a Bodhi server to create the update on. Ignored if staging is
-                       True.
-        user (unicode): The username to authenticate as.
-        password (unicode): The user's password.
-        staging (bool): Whether to use the staging server or not.
-        edit (bool): Set to True to edit an existing buildroot override.
-        kwargs (dict): Other keyword arguments passed to us by click.
+        url: The URL of a Bodhi server to create the update on. Ignored if staging is True.
+        user: The username to authenticate as.
+        password: The user's password.
+        staging: Whether to use the staging server or not.
+        edit: Set to True to edit an existing buildroot override.
```

Comment:
'openid_api' missing

Reply:
For this and the other missing things, I will make a second commit - teeeechnically they aren't really part of switching to type annotation, but I agree they should be documented.